apps/src/templates/SchoolNotFound.jsx

```diff
@@ -61,6 +63,9 @@ export default class SchoolNotFound extends Component {
     singleLineLayout: PropTypes.bool,
     showRequiredIndicators: PropTypes.bool,
     schoolNameLabel: PropTypes.string,
+    // Note: Google location search requires the following line to be present in the haml where this component is used:
+    // %script{type: "text/javascript", src: "https://maps.googleapis.com/maps/api/js?client=#{CDO.google_maps_client_id}&sensor=true&libraries=places,geometry&v=3.7"}
+    useGoogleLocationSearch: PropTypes.bool,
```

Comment:
I wonder if we could reasonably get the JavaScript that requires this to inject this script tag itself, instead of having this action-at-a-distance dependency on it.